.vsts-ci-no-sgx.yml

```diff
@@ -0,0 +1,109 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "ci/*"
+  paths:
+    exclude:
+      - 'README.md'
+      - 'CCF-TECHNICAL-REPORT.pdf'
+      - 'Dockerfile'
+      - 'Doxyfile'
+      - 'THIRD_PARTY_NOTICES.txt'
+      - 'getting_started/'
+      - 'sphinx/'
+      - '.circleci/'
+      - '.github/'
+      - '.vsts-gh-pages.yml'
+      - 'LICENSE'
+
+pr:
+  autoCancel: true
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - '*'
+    exclude:
```

Comment:
I wonder if we can use a variable for this. It'd be nice if we could.

Reply:
I think ideally it would be a template, but the template system only works within jobs. Looks like variables aren't supported in triggers either:

https://developercommunity.visualstudio.com/content/problem/380221/yaml-build-pipeline-path-triggers-do-not-support-v.html

Reply:
I think we might be able to use YAML references for this: https://confluence.atlassian.com/bitbucket/yaml-anchors-960154027.html

Reply:
Ah, I thought you meant we should de-duplicate this between `.vsts-ci.yml` and `.vsts-ci-no-sgx.yml`. I tried YAML references to de-duplicate within those files, but VSTS doesn't support them:

https://msr-gryffindor.visualstudio.com/COCO/_build/results?buildId=15836&view=results

Reply:
Ideally too yes, but within a file would have been a start. Anyway, thanks for trying.